codes/1963_cpp_wodnjs.cpp

```diff
@@ -0,0 +1,107 @@
+#include<iostream>
+#include<queue>
+#include<utility>
+
+using namespace std;
+bool not_prime[10000];
+int vis[10000];
+void prime_checker();
+void bfs(int, int);
+void clear_vis();
+int main() {
+	ios::sync_with_stdio(false);
+	cin.tie(nullptr);
+	cout.tie(nullptr);
+	int t_case;
+	cin >> t_case;
+	prime_checker();
+	for (int t = 0; t < t_case; t++)
+	{
+		int from, to;
+		cin >> from >> to;
+		bfs(from, to);
+		clear_vis();
+	}
+	return 0;
+}
+void prime_checker() {
+	for (int i = 2; i *i < 10000; i++)
+	{
+		if (not_prime[i] == false) 
+		{
+			for (int j = 2 * i; j < 10000; j += i)
```

Comment:
i * i부터 시작하는 것이 더 좋습니다.
```suggestion
			for (int j = i * i; j < 10000; j += i)
```